package.json

```diff
@@ -25,7 +25,7 @@
     "flow": "flow",
     "flow-typed": "flow-typed",
     "start": "npm run docs",
-    "docs": "npm run submodules:pull && BPK_BUILT_AT=$( date -u +%s ) webpack-dev-server --open",
```

Comment:
Why are we no longer pulling the submodules as part of the starting process?

Reply:
I'm restoring this I just tend to remove this line because I find it super annoying that it does this all the time

Reply:
Why not add a `docs:no-submodules-pull` command?

Reply:
Because I'm lazy